### PR TITLE
GH-485: adding advanced configurable parameters

### PR DESCRIPTION
**Pull request by bertllll**
I had to recreate this PR because did not update automatically for some reason.

### CODE REVIEW

**Review by FinsaasGH — APPROVED** [review]

File: node/src/sub_lib/neighborhood.rs [hunk 1]
```diff
@@ -31,10 +31,10 @@ use std::net::IpAddr;
...
 
 pub const DEFAULT_RATE_PACK: RatePack = RatePack {
-    routing_byte_rate: 100,
-    routing_service_rate: 10000,
-    exit_byte_rate: 101,
-    exit_service_rate: 10001,
+    routing_byte_rate: 1,
+    routing_service_rate: 100,
+    exit_byte_rate: 3,
+    exit_service_rate: 300,
```
Comment (kauri-hero) [inline node/src/sub_lib/neighborhood.rs]: After further review, I believe we should have the defaults as follows:

    routing_byte_rate: 1,
    routing_service_rate: 10,
    exit_byte_rate: 2,
    exit_service_rate: 20

File: node/src/sub_lib/accountant.rs [hunk 2]
```diff
@@ -19,11 +19,58 @@ lazy_static! {
     pub static ref TEMPORARY_CONSUMING_WALLET: Wallet = Wallet::from_str("0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF").expect("Internal error");
 }
 
-#[derive(Clone, PartialEq, Debug)]
-pub struct AccountantConfig {
-    pub payables_scan_interval: Duration,
-    pub receivables_scan_interval: Duration,
+lazy_static! {
+    pub static ref DEFAULT_PAYMENT_THRESHOLDS: PaymentThresholds = PaymentThresholds {
+        threshold_interval_sec: 2_592_000,
```
Comment (kauri-hero) [inline node/src/sub_lib/accountant.rs]: I misunderstood the variable earlier in my notes, so this needs to be corrected to:

threshold_interval_sec: 1800

File: node/src/sub_lib/accountant.rs [hunk 3]
```diff
@@ -19,11 +19,58 @@ lazy_static! {
...
+        debt_threshold_gwei: 10_000_000_000,
```
Comment (kauri-hero) [inline node/src/sub_lib/accountant.rs]: I made an error here for this value - it should be 1 MASQ:

debt_threshold_gwei: 1_000_000_000

File: node/src/sub_lib/accountant.rs [hunk 4]
```diff
@@ -94,19 +141,37 @@ pub struct FinancialStatisticsMessage {
...
             Wallet::from_str("0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF").expect("Internal error");
-
+        let payment_thresholds_expected = PaymentThresholds {
+            debt_threshold_gwei: 10_000_000_000,
+            maturity_threshold_sec: 1200,
+            payment_grace_period_sec: 1200,
+            permanent_debt_allowed_gwei: 500_000_000,
+            threshold_interval_sec: 2_592_000,
+            unban_below_gwei: 500_000_000,
+        };
```
Comment (kauri-hero) [inline node/src/sub_lib/accountant.rs]: if these defaults also need to be changed, please have them match the default corrections I made above

**Review by kauri-hero — CHANGES_REQUESTED** [review]
@bertllll 
I have updated the recommended defaults

File: node/src/sub_lib/accountant.rs (hunk 2 above)
Comment (bertllll) [reply node/src/sub_lib/accountant.rs]: I feel I will need to discuss this one further with you.

Comment (bertllll) [reply node/src/sub_lib/accountant.rs]: We ended up with 21600 being better.

**Review by FinsaasGH — APPROVED** [review]

**Review by FinsaasGH — APPROVED** [review]

**Review by FinsaasGH — APPROVED** [review]

**Review by kauri-hero — APPROVED** [review]

**Pull request merged**